bundles/org.openhab.binding.volvooncall/README.md

```diff
@@ -20,7 +20,7 @@ The binding has no configuration options itself, all configuration is done at 'T
 The 'VolvoOnCall API' bridge uses the owner's email address and password in order to access the VOC Remote API.
 This is the same email address and password as used in the VolvoOnCall smartphone app, that allows to remotely control your car(s).
 
-Once the bridge created, you will be able to launch discovery of the vehicles attached to it.
+Once the bridge created, you'll be able to launch discovery of the vehicles attached to it.
```

Comment:
Please, come back to previous way of writing. I think you need to rebase your local repository. This has been changed recently.

Reply:
Corrected